subsys/bluetooth/services/cts.c

```diff
@@ -0,0 +1,192 @@
+/** @file
+ *  @brief GATT Current Time Service
+ */
+
+/*
+ * Copyright (c) 2024 Croxel Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <stdbool.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/posix/time.h>
+
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/services/cts.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(cts, CONFIG_BT_CTS_LOG_LEVEL);
+
+#define BT_CTS_ATT_ERR_VALUES_IGNORED 0x80
+#define BT_CTS_FRACTION_256_MAX_VALUE 255
+
+static const struct bt_cts_cb *_cb;
```

Comment:
I'd just call this `cts_cb`. Underscore prefix can be useful e.g. for public struct members where you want to emphasize that it's private, but a static variable like this is implicitly private to the module it's in, so putting the prefix there seems unnecessary.